cpp/src/groupby/groupby.cu

```diff
@@ -124,6 +124,14 @@ struct empty_column_constructor {
     }
     if constexpr (k == aggregation::Kind::MERGE_HISTOGRAM) { return empty_like(values); }
 
+    if constexpr (k == aggregation::Kind::SUM_ANSI) {
+      // SUM_ANSI returns a struct with sum (int64_t) and overflow (bool) children
+      std::vector<std::unique_ptr<cudf::column>> children;
+      children.push_back(make_empty_column(cudf::data_type{cudf::type_id::INT64}));
+      children.push_back(make_empty_column(cudf::data_type{cudf::type_id::BOOL8}));
+      return make_structs_column(0, std::move(children), 0, {}, stream, mr);
```

Comment:
What input types are supported for `sum_ansi`? It seems only signed integers. However, I don't see any exceptions reported if an unsupported input type is provided?

Reply:
Currently, it only supports `int64`, and I’ve added a test to ensure it throws an error if a different type is passed to this groupby.